비밀지도.js

```diff
@@ -0,0 +1,15 @@
+function solution(n, arr1, arr2) {
+    const bins = [];
+    const answer = [];
+    
+    for (let i = 0; i < n; i++) {
```

Comment:
```suggestion
    for (let i = 0; i < n; i += 1) {
```
[no-plusplus 규칙](https://eslint.org/docs/rules/no-plusplus)